app/services/data_sources/json_notification_data_source.rb

```diff
@@ -37,8 +37,7 @@ def notifications
       JSONProjectDataSource.projects_file_semaphore.synchronize do
         return unless File.exist?(notifications_file_path)
 
-        return JSON.parse(File.read(notifications_file_path))
-                   .map(&Notification.method(:from_json!))
+        return JSON.parse(File.read(notifications_file_path)).map(&Notification.method(:from_json!))
```

Comment:
Oh noes, I like having `.map` on a different line, much functional, wow 😢

Reply:
no worries - I'll revert this bit back